pkg/gateway/vald/handler/grpc/handler.go

```diff
@@ -217,44 +209,44 @@ func (s *server) search(ctx context.Context, cfg *payload.Search_Config,
 			}
 			return res, nil
 		case dist := <-dch:
-			if len(res.GetResults()) >= num {
-				if dist.GetDistance() < math.Float32frombits(atomic.LoadUint32(&maxDist)) {
-					atomic.StoreUint32(&maxDist, math.Float32bits(dist.GetDistance()))
-				} else {
-					continue
-				}
+			pos := len(res.GetResults())
```

Comment:
`pos` is misleading name. please rename it like `resultLen`, `resLen` or something.

Reply:
position is misreading??

Reply:
what is the position of? here, this variable is just used for L213 and L216.
they should be treated as a different variable from the variable on L226 (that is the "true" `pos` variable).

Reply:
position of result end index

Reply:
you should name it as like that, i think.

Reply:
hmm, anything is okay, it's not important

Reply:
I'll change it and copy it to pos